docs/source/ecf.rst

```diff
@@ -540,16 +497,48 @@ The path to the directory in which to output the Stage 5 JWST data and plots.
 Stage 5 Fit Parameters
 ----------------------
 
-This file describes the transit/eclipse parameters and their prior distributions. Each line describes a new parameter, with the following basic format:
+.. warning::
+   The Stage 5 fit parameter file has the file extension ``.epf``, not ``.ecf``. These have different formats, and are not interchangeable.
+
+This file describes the transit/eclipse and systematics parameters and their prior distributions. Each line describes a new parameter, with the following basic format:
 
 ``Name    Value    Free    PriorPar1    PriorPar2    PriorType``
 
-The ``PriorType`` can be U (Uniform), LU (Log Uniform), or N (Normal). If U/LU, then ``PriorPar1`` and ``PriorPar2`` are the lower and upper limits of the prior distribution. If N, then ``PriorPar1`` is the mean and ``PriorPar2`` is the stadard deviation of the Gaussian prior.
+``Name`` defines the specific parameter being fit for. Available options are:
+   - Transit and Eclipse Parameters
+      - ``rp`` - planet radius, for the transit models.
+      - ``fp`` - planet/star flux ratio, for the eclipse models.
+   - Orbital Parameters
+      - ``per`` - orbital period (in days)
+      - ``t0`` - transit time (in days)
+      - ``time_offset`` - (optional), the absolute time offset of your time-series data (in days)
+      - ``inc`` - orbital inclination (in degrees)
+      - ``a`` - a/R*, the ratio of the semimajor axis to the stellar radius
+      - ``ecc`` - orbital eccentricity
+      - ``w`` - argument of periapsis (degrees)
+   - Phase Curve Parameters - the phase curve model allows for the addition of up to four sinusoids into a single phase curve
+      - ``AmpCos1`` - Amplitude of the first cosine 
+      - ``AmpSin1`` - Amplitude of the first sine
+      - ``AmpCos2`` - Amplitude of the second cosine
+      - ``AmpSin2`` - Amplitude of the second sine
+   - Limb Darkening Parameters
+      - ``limb_dark`` - The limb darkening model to be used. Options are: ``['uniform', 'linear', 'quadratic', 'kipping2013', 'square-root', 'logarithmic', 'exponential', '4-parameter']``
+      - ``u1`` - The first limb darkening parameter
+      - ``u2`` - The second limb darkening parameter.
+      Further limb darkening parameters can be specified as needed by the specified model.
```

Comment:
I would document the limb darkening parameters differently. I would make it clearer that there are no u parameters for 'uniform', just u1 for 'linear', u1-2 for 'quadratic', u1-2 for 'kipping2013', 'square-root', 'logarithmic', 'exponential', and u1-4 for '4-parameter'

Reply:
Done.